src/app-bridge/app-bridge.ts

```diff
@@ -94,11 +95,24 @@ const getDomainFromUrl = () =>
 const getSaleorApiUrlFromUrl = () =>
   new URL(window.location.href).searchParams.get(AppIframeParams.SALEOR_API_URL) || "";
 
+const getThemeFromUrl = () => {
+  const value = new URL(window.location.href).searchParams.get(AppIframeParams.THEME);
+
+  switch (value) {
+    case "dark":
+    case "light":
+      return value;
+    default:
+      return undefined;
+  }
+};
+
 const getDefaultOptions = (): AppBridgeOptions => ({
   targetDomain: getDomainFromUrl(),
   saleorApiUrl: getSaleorApiUrlFromUrl(),
   initialLocale: getLocaleFromUrl() ?? "en",
-  autoNotifyReady: true,
+  autoNotifyReady: false,
```

Comment:
changing to false, because Dashboard hasn't implemented it yet, so its throwing unknown action